generated/niscope/niscope/session.py

```diff
@@ -1997,6 +1996,164 @@ def fetch(self, num_samples=None, relative_to=enums.FetchRelativeTo.PRETRIGGER,
 
         return wfm_info
 
+    @ivi_synchronized
+    def fetch_array_measurement(self, array_meas_function, timeout=hightime.timedelta(seconds=5.0), other_channel=None):
```

Comment:
`measWfmSize` is an input parameter in [C](http://zone.ni.com/reference/en-XX/help/370592AB-01/scopefunc/cviniscope_fetcharraymeasurement/) and [LabVIEW](https://www.ni.com/documentation/en/ni-scope/latest/scopenodes/fetch-measurement-array-multi-1d-cluster/) APIs. Why is it not present here?

Reply:
I believe it's handled internally rather than manually, as it's never generated as a parameter for _fetch_array_measurement. See [these lines of code](https://github.com/shilohbeeler-NI/nimi-python/blob/d6792db8d5188eb7f488084a68a8f4dd4654a1be/generated/niscope/niscope/session.py#L2576).

Reply:
I'm not sure how much vetting `_fetch_array_measurement` has gone through since it's a private method. Can you please check with NI-SCOPE experts once?